tasks/main.yml

```diff
@@ -52,6 +52,7 @@
         name: libselinux-python
         state: present
       when: ansible_os_family == "RedHat" and ansible_distribution_major_version >= 7
+      ignore_errors: true # The when statement above is not reliable on hLinux
```

Comment:
We found the root cause (a bug in our fork ansible-role-provision-vm caused fact checking to run on the CentOS hypervisor, but then not on the hLinux guest.